Documentation/DocuBlocks/Rest/Views/patch_api_view_properties_iresearch.md

```diff
@@ -72,32 +72,25 @@ Background:
 The segment candidates for the "consolidation" operation are selected based
 upon several possible configurable formulas as defined by their types.
 The currently supported types are (default: "bytes_accum"):
-- *bytes*: consolidate if and only if:
+- *bytes*: consolidate if and only if ({threshold} range `[0.0, 1.0]`):
   {threshold} > segment_bytes / (all_segment_bytes / number_of_segments)
   i.e. the candidate segment byte size is less that the average segment
        byte size multiplied by the {threshold}
-- *bytes_accum*: consolidate if and only if:
+- *bytes_accum*: consolidate if and only if ({threshold} range `[0.0, 1.0]`):
   {threshold} > (segment_bytes + sum_of_merge_candidate_segment_bytes) / all_segment_bytes
   i.e. the sum of all candidate segment byte size is less than the total
        segment byte size multiplied by the {threshold}
-- *count*: consolidate if and only if:
+- *count*: consolidate if and only if ({threshold} range `[0.0, 1.0]`):
   {threshold} > segment_docs{valid} / (all_segment_docs{valid} / number_of_segments)
   i.e. the candidate segment non-deleted document count is less that the
        average segment non-deleted document count size multiplied by the
        {threshold}
-- *fill*: consolidate if and only if:
+- *fill*: consolidate if and only if ({threshold} range `[0.0, 1.0]`):
   {threshold} > #segment_docs{valid} / (#segment_docs{valid} + number_of_segment_docs{removed})
   i.e. the candidate segment valid document count is less that the average
        segment total document count multiplied by the {threshold}
-
-@RESTSTRUCT{segmentThreshold,post_api_view_props_consolidation,integer,optional,uint64}
-Apply the "consolidation" operation if and only if (default: 300):
-{segmentThreshold} < number_of_segments
-
-@RESTSTRUCT{threshold,post_api_view_props_consolidation,number,optional,float}
-Select a given segment for "consolidation" if and only if the formula based
-on *type* (as defined above) evaluates to true, valid value range [0.0, 1.0]
-(default: 0.85)
+- *tier*: consolidate based on segment byte size and live document count
+          as dicated by the customization attributes.
```

Comment:
Do not indent text like this! With list indention plus 4 or more spaces, it will create a code block.

Should be (also fixed typo dicated --> dictated):
```md
- *tier*: consolidate based on segment byte size and live document count
  as dictated by the customization attributes.
```

Reply:
this is a backport of 'devel'